cmd/skaffold/app/cmd/cmd.go

```diff
@@ -103,6 +108,15 @@ func NewSkaffoldCommand(out, err io.Writer) *cobra.Command {
 			return nil
 		},
 		PersistentPostRun: func(cmd *cobra.Command, args []string) {
+			if !isHouseKeepingMessagesAllowed(cmd) {
```

Comment:
so, we should also make this check in L95 and not call the `go func` which makes these calls.

Reply:
makes sense